db/queries.py

```diff
@@ -3,44 +3,61 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(title__startswith="Untitled"))
 
 
-def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+def get_users_who_sent_messages_in_2015() -> list[User]:
+    return list(User.objects.filter(message__sent__year="2015").
+                values_list("first_name", "last_name").distinct())
```

Comment:
why do you use `.distinct()` here?

Reply:
Thanks! Sure, it's useless.
Most likely, the sample was taken from the previous task of searching by date and there was distinct(), and then forgot to remove it.